Mathlib/Tactic/GuardHypNums.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright (c) 2022 Robert Y. Lewis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robert Y. Lewis
+-/
+
+import Lean
+
+/-!
+A tactic stub file for the `guard_hyp_nums` tactic.
+-/
+
+open Lean Meta Elab Tactic
+
+/-- `localContextLength` returns the number of hypotheses in the local context,
+including hidden and auxiliary hypotheses. -/
+def localContextLength : TacticM Nat := do
```

Comment:
This seems more like `LocalContext.size`. Are you sure there isn't such a function already?

Reply:
I'm not sure, but Leo suggested this. `LocalContext.decls` contains irrelevant `none`s that we don't want to count.

Reply:
okay, but it should still be a function called `LocalContext.size` IMO

Reply:
PR'd to core as https://github.com/leanprover/lean4/pull/1323

Reply:
Done now in https://github.com/leanprover-community/mathlib4/pull/345